src/db/order-repository.ts

```diff
@@ -0,0 +1,86 @@
+import { Logger } from '../logger';
+import { InitDBResponse } from './db';
+import { from, Observable } from 'rxjs';
+import { Order, Trade } from 'ccxt';
+import { OrderAttributes, OrderInstance } from './order';
+import { map, mergeMap } from 'rxjs/operators';
+import { TradeAttributes } from './trade';
+
+type SaveOrderParams = {
+  order: Order;
+  models: InitDBResponse;
+  logger: Logger;
+};
+
+const saveOrder$ = ({
+  order,
+  models,
+  logger,
+}: SaveOrderParams): Observable<OrderInstance> => {
+  logger.trace(`Saving order ${JSON.stringify(order)} into database`);
+
+  const arbyOrder = convertToArbyOrder(order);
+  const arbyTrades = convertToArbyTrades(order.trades, order.id);
+  // TODO async
```

Comment:
TODO?

Reply:
Yes actually saving into database is not a big deal, but may interrupt the responsiveness of arby, so I added a TODO there to take care later and make save operation async with schedulers of rxjs, wdyt?